README.md

```diff
@@ -344,6 +345,27 @@ If you need to invoke your Cloud Run applications periodically, use
 [Google Cloud Scheduler](https://cloud.google.com/scheduler/). It can make a
 request to your application’s specific URL at an interval you specify.
 
+### Can I mount storage volumes or disks on Cloud Run?
+
+Cloud Run currently doesn’t offer a way to bind mount additional storage volumes
```

Comment:
Can you also capture this information in the first party docs? 
For example on https://cloud.google.com/run/docs/using-gcp-services#services_not_yet_supported
and https://cloud.google.com/run/docs/issues

Updating the FAQ should come after the first party doc has been fixed.

Reply:
Will do. That said, in the future I'll be passing the snippets added here to TWs to write it better. It takes 3 minutes for me to write it here, ~30 minutes on internal tooling, staging, sending for reviews, back and forths etc. :)